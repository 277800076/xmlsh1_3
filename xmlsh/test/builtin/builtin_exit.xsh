# Test exit
# If called with 0 arguments then test
# else exit with arg

if [ $# -eq 0 ] ; then 
	$0 1
	echo return value $?
	$0 2
	echo return value $?
	exit 0
else 
	exit $1
fi
