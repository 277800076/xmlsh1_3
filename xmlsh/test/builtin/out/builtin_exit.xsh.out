return value 1
return value 2
